server/services/prisonerService.js

```diff
@@ -70,17 +70,11 @@ function createPrisonerService(nomisClientBuilder) {
     return [merge(coms[0], { deliusId: id.identifierValue })]
   }
 
-  async function getPrisonerImage(imageId, token) {
-    try {
-      logger.info(`getPrisonerImage: ${imageId}`)
+  function getPrisonerImage(imageId, token) {
```

Comment:
do you prefer `function` over `const getPrisonerImage` - just wondered why not changed to arrow syntax?